### PR TITLE
resource/alicloud_tag_policy: Added user_type compute and test fault tolerance

### PR DESCRIPTION
**Pull request by fuliu-zln**

### CODE REVIEW

File: alicloud/service_alicloud_tag.go [hunk 1]
```diff
@@ -91,6 +91,12 @@ func (s *TagService) DescribeTagPolicy(id string) (object map[string]interface{}
 		return nil
 	})
 	addDebug(action, response, request)
+	if err != nil {
+		if IsExpectedErrors(err, []string{"InvalidParameter.PolicyId"}) {
+			return object, WrapErrorf(Error(GetNotFoundMessage("Tag:policy", id)), NotFoundWithResponse, response)
```
Comment (MrWolong) [inline alicloud/service_alicloud_tag.go:96]: Tag:Policy

File: alicloud/resource_alicloud_tag_policy.go [hunk 2]
```diff
@@ -44,6 +44,7 @@ func resourceAlicloudTagPolicy() *schema.Resource {
 				Optional:     true,
```
Comment (MrWolong) [inline alicloud/resource_alicloud_tag_policy.go:44]: 标题改一下：resource/alicloud_tag_policy: Added `user_type` compute and test fault tolerance

File: alicloud/service_alicloud_tag.go (hunk 1 above)
Comment (fuliu-zln) [reply alicloud/service_alicloud_tag.go:96]: done

File: alicloud/resource_alicloud_tag_policy.go (hunk 2 above)
Comment (fuliu-zln) [reply alicloud/resource_alicloud_tag_policy.go:44]: done

File: alicloud/resource_alicloud_tag_policy_test.go [hunk 3]
```diff
@@ -84,6 +84,48 @@ func TestAccAlicloudTagPolicy_basic(t *testing.T) {
...
+			testAccPreCheck(t)
+		},
+		IDRefreshName: resourceId,
+		Providers:     testAccProviders,
+		CheckDestroy:  rac.checkResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccConfig(map[string]interface{}{
+					"policy_content": `{\"tags\":{\"CostCenter\":{\"tag_value\":{\"@@assign\":[\"Beijing\",\"Shanghai\"]},\"tag_key\":{\"@@assign\":\"CostCenter\"}}}}`,
+					"policy_name":    name,
```
Comment (MrWolong) [inline alicloud/resource_alicloud_tag_policy_test.go:111]: policy_name是可改的吗？可改的话，对可改字段进行修改测试；不可改，对字段加forcenew

Comment (fuliu-zln) [reply alicloud/resource_alicloud_tag_policy_test.go:111]: 已经有该字段的修改测试了

**Review by xiaozhu36 — APPROVED** [review]

**Pull request merged**